src/components/Notification/Notification.js

```diff
@@ -181,15 +172,64 @@ export class ToastNotification extends Component {
       hideCloseButton,
       ...other
     } = this.props;
-
     const classes = classNames(
       'bx--toast-notification',
       { [`bx--toast-notification--${this.props.kind}`]: this.props.kind },
       className
     );
+    const a11yWarningIcon = (
+      <svg
```

Comment:
Probably we'd make it a separate React component in a separate module?

Reply:
sure, do we want to keep it within the `src/components/Notification` directory then?